src/sass/domain/engagement-status.scss

```diff
@@ -0,0 +1,12 @@
+.engagement-status__icon {
+  .icon {
+    font-size: 14px;
+  }
+
+  &.active { color: color(green); }
```

Comment:
I _think_ we aren't going to end up needing to support all of these statuses.. but I mis-typed kim's email so waiting the hear back..  can probably just merge as is and clean up later if needed.  but dropping a note here.